### PR TITLE
Rename "CI Integrations" to "Notifications"

### PR DESCRIPTION
**Pull request by underyx**
Made this change because CI Integrations sounds like it's about which CI providers Semgrep CI can run in. We already use "Notifications" on the main Semgrep CI docs page.

### CODE REVIEW

File: docs/notifications.md [hunk 1]
```diff
@@ -51,7 +66,11 @@ If you are using Github Actions to run Semgrep, no extra changes are needed to g
 - `SEMGREP_PR_ID` is set to the PR number of the pull request on Github (e.g. `2900`)
 - `SEMGREP_REPO_NAME` is set to the repo name (e.g. `returntocorp/semgrep`)
 
-### Webhooks
+## GitLab merge request comments
+
+Merge request comments on GitLab are coming soon. [Sign up for the beta here.](https://go.r2c.dev/join-gitlab-beta)
```
Comment (pabloest) [inline docs/notifications.md:71]:
```suggestion
Interested in GitLab merge request comments? [Sign up for the beta here.](https://go.r2c.dev/join-gitlab-beta)
```

File: docs/notifications.md [hunk 2]
```diff
@@ -21,18 +38,16 @@ Use the 'Test' button to send a test notification and ensure that your channel i
...
 
 To receive email notifications about Semgrep findings on pull requests and code pushes, visit [Dashboard > Integrations](https://semgrep.dev/manage/integrations) and select 'Add integration' or 'Setup First Integration,' and then choose 'Email'. Enter your email address, give the channel a name of your choosing, and then click 'Save'.
 
 On each scan that has at least one finding, you will receive one email from Semgrep with a summary of all of the findings from that scan.
 
-### Pull request comments
+## GitHub pull request comments
 
-<!-- prettier-ignore-start -->
 !!! info
```
Comment (pabloest) [inline docs/notifications.md:49]: Do we still need this info box given that the subheading directly above already includes “GitHub?”

File: docs/notifications.md [hunk 3]
```diff
@@ -51,7 +66,11 @@ If you are using Github Actions to run Semgrep, no extra changes are needed to g
...
 - `SEMGREP_REPO_NAME` is set to the repo name (e.g. `returntocorp/semgrep`)
 
-### Webhooks
+## GitLab merge request comments
+
+Merge request comments on GitLab are coming soon. [Sign up for the beta here.](https://go.r2c.dev/join-gitlab-beta)
+
+## Webhooks
 
 Webhook notifications are a paid feature in our Semgrep Team tier.
```
Comment (pabloest) [inline docs/notifications.md:75]:
```suggestion
Webhook notifications are a paid feature in the Semgrep Team tier.
```

**Review by pabloest — APPROVED** [review]
Overall looks great, Bence.

Please also add a redirect from integrations → notifications by adding a line at the bottom of `mkdocs.yml` inside `redirect_maps`:

```
"integrations.md": "notifications.md"
```

File: docs/notifications.md (hunk 2 above)
Comment (underyx) [reply docs/notifications.md:49]: good point!

**Pull request merged**